packages/tooling/cfp-template/template/src/components/welcome/welcome.ts

```diff
@@ -0,0 +1,22 @@
+import { attr, customElement, FASTElement } from "@microsoft/fast-element";
```

Comment:
I think we should simplify the initial web component by removing as much of the CSS as possible and getting rid of the `theme` part. I think that might go against the adaptive UI approach that the rest of the libraries support. No need to complicate the starter template with that IMO.

Reply:
(This is the only real reason for blocking. The rest we can deal with in a future update.)

Reply:
Well, the only issue with that is it removes the "thing I can write a test against" reasoning for having it in there. Do you have an alternate suggestion? Otherwise I suppose I could remove it and the tests just check for the presence of the component.